### ISSUE TITLE
Add instructions to verify checksum of exiftool bins included with the app

### ISSUE THREAD

**Issue by szTheory** [szTheory/exifcleaner#16]
Obviously I can trust myself since I'm the one who pulled the exiftool binaries and put them in the app. But for other users especially advanced ones they might want to confirm that the binaries haven't been tampered with for security reasons. It should be sufficient to provide instructions in the README for how to generate checksum of the binary (there are two, one for linux/mac and one for Windows) and compare it against what is listed on the [official ExifTool website](https://exiftool.org/).

If someone wants to do the legwork on this feel free. Otherwise I hope to get around to it soon.

**Comment by szTheory** [comment]
The binaries come from the `.resources` dir within the project which I pulled from the official website about 3 months ago when I first built the app. I noticed they weren't being included in source control due to a wildcard in the .gitignore template, so I added an override to include them in e3ae0bdeddbc0a9e521c645a7c451d8a88f0e3a2

I'm also thinking about how frequently to update the binaries. Maybe every year or so. It's nice to get updates and bugfixes, but I don't want to constantly release new versions of the GUI app to keep up with every update of the command line tool. It's already pretty mature and the benefits will be marginal to frequent updates. It could also create an unnecessarily large support burden if there is an issue with a specific `exiftool` that is constantly changing.
